### PR TITLE
feat(addon-charts): `LineChart` add custom dot

### LINKED ISSUE #1578: 🚀 - Ability to view dot on line-chart only for hovered point
### Which @taiga-ui/* package(s) are relevant/releated to the feature request?

addon-charts

### Description

We need single dot in line-charts, which will show only with hover + hint.

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?
dots are not customizable

Closes #1578

## What is the new behavior?
dots are customizable

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/charts/line-chart/examples/6/index.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.axes {
+    height: 12.5rem;
+    width: 25rem;
+    color: #bc71c9;
+}
+
+.icon {
+    background: white;
```
Comment (nsbarsukov) [inline projects/demo/src/modules/charts/line-chart/examples/6/index.less:8]: bad for dark mode

dark-mode

___

Is `--tui-base-01` an appropriate alternative ?

File: projects/demo/src/modules/charts/line-chart/examples/6/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        [value]="value"
+        [xStringify]="stringify"
+        [yStringify]="stringify"
+    ></tui-line-chart>
+</tui-axes>
+
+<ng-template
+    #dot
+    let-point
+    let-hovered="hovered"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/charts/line-chart/examples/6/index.html:21]: Can we use `(tuiHoveredChange)` (on svg) instead ?

Or it does not work ?

File: projects/demo/src/modules/charts/line-chart/examples/6/index.less (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/charts/line-chart/examples/6/index.less]: thanks! fixed

File: projects/demo/src/modules/charts/line-chart/examples/6/index.html (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/charts/line-chart/examples/6/index.html:21]: We need to show icons on hover on entire column, not only on icon, thats why I use hovered from context (it means hovered column)

File: projects/addon-charts/components/line-chart/line-chart.component.ts [hunk 3]
```diff
@@ -88,6 +88,10 @@ export class TuiLineChartComponent {
     @tuiDefaultProp()
     dots = false;
 
+    @Input()
+    @tuiDefaultProp()
+    dotContent: PolymorpheusContent = '';
```
Comment (waterplea) [inline projects/addon-charts/components/line-chart/line-chart.component.ts:93]: Maybe we can just change type of `dots` to `boolean | PolymorpheusContent`? Also make sure to add context type.

**Review by waterplea — CHANGES_REQUESTED** [review]

**Pull request closed without merging**